parlai/crowdsourcing/tasks/pairwise_per_turn_eval/hydra_configs/conf/example_model_comparison.yaml

```diff
@@ -0,0 +1,31 @@
+#@package _global_
+defaults:
+  - /mephisto/blueprint: per_turn_eval_blueprint
+  - /mephisto/architect: local
+  - /mephisto/provider: mock
+mephisto:
+  blueprint:
+    annotations_config_path: ${task_dir}/task_config/annotations_config.json
+    onboarding_qualification: per_turn_eval_onboarding__engaging
+    block_qualification: per_turn_eval_${current_time}_block  # Use the current time in the block, like with Acute-Evals
+    chat_data_folder: # TODO: Set this to your data directory
```

Comment:
I'd set this to `${task_dir}/chat_data/` so that the user can just call `run.py` as-is with no other commands and have everything just work